instruqt/delivering-as-an-appliance/06-completing-the-install/assignment.md

```diff
@@ -0,0 +1,219 @@
+tabs:
+- id: i1b6db1hruhx
+  title: Admin Console
+  type: website
+  url: http://node.${_SANDBOX_ID}.instruqt.io:30000
+  new_window: true
+- id: omi9odffx8es
+  title: Vendor Portal
+  type: website
+  url: https://vendor.replicated.com
+  new_window: true
+difficulty: basic
+timelimit: 1200
+---
+
+We're part way through the installation of SlackerNews as a Kubernetes
+Appliance. We've run the SlackerNews installer which has create a single-node
+Kubernetes cluster running all of the infrastructure components of the
+Replicated Embedded Cluster. The next step is to complete the install using
+one of those components, the Admin Console.
+
+About the Admin Console
+=======================
+
+The Replicated Admin Console is a web-based interface that plays two major
+roles:
+
+1. It wraps the Helm command with a user-friendly interface that guides the
+   user through the installation process.
+2. It offers a GUI for "Day Two" operations like checking for (and installing)
+   updates, collecting support bundles, and managing cluster nodes.
+
+We're going to focus primarily on the first role for now.
+
+Completing the Install
+=====================
+
+To help us complete our installation, the Admin Console will guide us a
+through a few steps, starting with configuring its certificate. Since it's
+not possible to provide a "safe" certificate to the Admin Console out of the
+box, the process starts with a warning and instructions for the user to
+validate the self-signed certificate the installer created.
+
+![Explaining the Initial Certificate to Your Customer](../assets/certificate-warning.png)
+
+The next step will ask your customer to configure the permanent certificate
+for the Admin Console. They can choose between a self-signed certificate or
+uploading one signed by a certificate authority. I recommend that they use a
+signed certificate, this is something you may also want to suggest in your
+installation documentation.
+
+For the lab, though, we'll stick with the self-signed certificate. When you
+click the "Continue to Setup" button, you'll get a warning about the initial
+self-signed certificate. Assuming you accept the risk, you'll get a form that
+allows you to set up the SlackerNews Admin Console certificate. Keep it set on
+"self-signed" and click "Continue".
+
+![Configuring the Admin Console Certificate](../assets/certificate-configuration.png)
+
+You'll be asked to log into the Admin Console on the next page. The password
+for the Admin Console is the password you specified in the last step. It will
+be `[[ Instruqt-Var key="ADMIN_CONSOLE_PASSWORD" hostname="node" ]]` if you use the suggestion
+from the lab text or skipped ahead to this step.
+
+### Adding Nodes to the Cluster
+
+The first thing you'll see when you log in to the Admin Console is the Node
+Management screen. This is where you can add additional nodes to the
+SlackerNews Embedded Cluster. Node management is the first step in the install
```

Comment:
```suggestion
cluster embedded with SlackerNews. Node management is the first step in the install
```
Suggestion to avoid using capitalized Embedded Cluster when talking about the cluster itself (versus as the proper name of the installer)